libraries/lib-ffmpeg-support/wrappers/AVCodecContextWrapper.cpp

```diff
@@ -131,11 +146,9 @@ AVCodecContextWrapper::DecodeAudioPacket(const AVPacketWrapper* packet)
             std::copy(frameData, frameData + frameSize, data.begin());
          }
 
-         if (!packetCopy->OffsetPacket(bytesDecoded))
-            packetSizeLeft = 0;
-         else
-            packetSizeLeft = packetCopy->GetSize();
+         packetCopy->OffsetPacket(bytesDecoded);
```

Comment:
A fix to OffsetPacket was required after this change. I'm not sure if this code is really safe, though